packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -519,9 +519,14 @@ class HotRunner extends ResidentRunner {
     await _evictDirtyAssets();
     printTrace('Reassembling application');
     bool reassembleAndScheduleErrors = false;
+    bool reassembleTimeout = false;
```

Comment:
nit: `reassemleTimedOut`? "Timeout" sounds more like the _length_ of the timeout.